klasses/api_test.py

```diff
@@ -24,7 +25,7 @@
 
 
 @pytest.fixture()
-def setup_test_data():
+def setup_test_data(mocker):
```

Comment:
I called `setup_test_data` because the same kind of setup is done in the `setUpTestData` classmethod in the unittest class based tests.
To me it makes more sense to be consistent in this sense.